docs/roadmap/index.md

```diff
@@ -55,9 +55,9 @@ Non-Goals
 
 Tasks
 
-- [ ]  [Design the image scanning and automation API](https://github.com/fluxcd/toolkit/discussions/107)
+- [x]  [Design the image scanning and automation API](https://github.com/fluxcd/toolkit/discussions/107)
 - [ ]  Implement an image scanning controller
-- [ ]  Design the automation component
+- [x]  Design the automation component
```

Comment:
```suggestion
- [x]  <span style="color:grey">Design the automation component</span>
```

Reply:
Should the other "done" item have this too? I'll adjust it